src/plugins/WebOfTrust/util/TickerDelayedBackgroundJob.java

```diff
@@ -0,0 +1,363 @@
+package plugins.WebOfTrust.util;
+
+import freenet.node.FastRunnable;
+import freenet.node.PrioRunnable;
+import freenet.support.Executor;
+import freenet.support.Ticker;
+import freenet.support.io.NativeThread;
+
+/**
+ * A {@link DelayedBackgroundJob} implementation that uses a {@link Ticker} for scheduling.
+ *
+ * @author bertm
+ * @see TickerDelayedBackgroundJobFactory
+ */
+public class TickerDelayedBackgroundJob implements DelayedBackgroundJob {
```

Comment:
(Have explained this at the other comment)

To speed up the release, I've fixed this myself already.

-> Done.